### ISSUE TITLE
Update confetti asset on /connect page

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#910]
The confetti asset is very overwhelming on the connect page because it's so large. There are also a number of confetti pieces that overlap other confetti, and, in general, the page feels very busy.

<img width="1276" alt="screen shot 2017-09-24 at 7 19 23 pm" src="https://user-images.githubusercontent.com/5550969/30788166-56606f74-a15d-11e7-8700-8142258a6294.png">

**Comment by theandrewbriggs** [comment]
This is live on production.

**Comment by amynicole** [comment]
I think we can actually remove the confetti from this page and leave it more minimal.

**Comment by smarziano** [comment]
I agree that we can remove the confetti on this page. It still looks too busy.
